### ISSUE TITLE
"Un"mirror Preview

### ISSUE THREAD

**Issue by ax3l** [palavatv/palava#41]
**Is your feature request related to a problem? Please describe.**

When using the rear-side camera of a smartphone or streaming from a Linux loopback video device the preview video should not be mirrored.

**Describe the solution you'd like**
Potentially add a button to "un"mirror the preview :)
Maybe there is also an idiomatic way to find a good default for a video source (front-facing or not).

**Describe alternatives you've considered**
I tried not to watch the preview.

**Additional context**
The same thing will be needed for screen sharing #17

**Comment by janlelis** [comment]
Right. Automatic detection would be awesome, but we can't be sure to always detect this properly. I am thinking in the direction of an option in the join-room dialog…

**Comment by farao** [comment]
or a flip button on the video?

**Comment by janlelis** [comment]
> or a flip button on the video?

I am rather hesitant to add new buttons, and it is something you usually don't need to switch *during* the call

**Comment by ax3l** [comment]
But one has to switch it during a call with a smartphone: "hey, let me show you my new flat real quick" - switch to back camera - "ui, confusing" - show around - switch back to front camera.
